### PR TITLE
refactor: implemented new user profile page design

### LINKED ISSUE #379: Redesign User Public Profile

### PR DESCRIPTION
**Pull request by boandriy**
implemented new user profile page design

closes #379

### CODE REVIEW

**Review by chriswhong — CHANGES_REQUESTED** [review]
Looks like we still need to remove the "following" tab and add the user's avatar image in the header.

**Review by chriswhong — APPROVED** [review]
Pushed some styling tweaks to get things closer to the mockups.  LGTM unless there are changes requested by engineering.

File: src/features/userProfile/UserProfile.tsx [hunk 1]
```diff
@@ -96,12 +95,13 @@ const UserProfile: React.FC<UserProfileProps> = ({ path = '/' }) => {
...
-      numberDecorator: paginatedFollowingResults.pageInfo.resultCount,
-      selected: path === '/following'
-    }
+    // TODO (boandriy): Hiding temporarily to match new design
+    // {
+    //   name: 'Following',
+    //   link: `/${usernameParam}/following`,
+    //   numberDecorator: paginatedFollowingResults.pageInfo.resultCount,
+    //   selected: path === '/following'
+    // }
```
Comment (ramfox) [inline src/features/userProfile/UserProfile.tsx:104]: Can you write in the TODO when it should be commented back in, or remove the commented code

**Review by ramfox — APPROVED** [review]
LGTM! Just one question about some commented out code and then it's good to merge!

**Pull request merged**